web/components/promtools-app.js

```diff
@@ -0,0 +1,152 @@
+import {css, html, LitElement} from 'https://unpkg.com/lit-element@2.3.1/lit-element.js?module';
+import './promtools-form.js';
+import './promtools-output.js';
+
+class App extends LitElement {
+    static get properties() {
+        return {
+            loading: {type: Boolean},
```

Comment:
I believe the type property is only valuable if you're doing attribute reflection 
I think Justin Fagnani suggested just doing:
`loading: { attribute: false }` in these cases. See https://lit-element.polymer-project.org/guide/properties --> type for more info

Reply:
Nice! Yes, I wasn't sure where to put these props so they still invoke a re-render :+1: